kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/CreateAclBatchAction.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2022 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.resources.v3;
+
+import static java.util.Objects.requireNonNull;
+
+import io.confluent.kafkarest.Errors;
+import io.confluent.kafkarest.controllers.AclManager;
+import io.confluent.kafkarest.entities.Acl;
+import io.confluent.kafkarest.entities.v3.CreateAclBatchRequest;
+import io.confluent.kafkarest.extension.ResourceAccesslistFeature.ResourceName;
+import io.confluent.kafkarest.resources.AsyncResponses;
+import io.confluent.rest.annotations.PerformanceMetric;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import javax.inject.Provider;
+import javax.validation.Valid;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.container.AsyncResponse;
+import javax.ws.rs.container.Suspended;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/v3/clusters/{clusterId}/acls:batch")
+@ResourceName("api.v3.acls.*")
+public final class CreateAclBatchAction {
+
+  private final Provider<AclManager> aclManager;
+
+  @Inject
+  public CreateAclBatchAction(Provider<AclManager> aclManager) {
+    this.aclManager = requireNonNull(aclManager);
+  }
+
+  @POST
+  @Consumes(MediaType.APPLICATION_JSON)
+  @Produces(MediaType.APPLICATION_JSON)
+  @PerformanceMetric("v3.acls.create")
+  @ResourceName("api.v3.acls.create")
```

Comment:
I think these should clearly indicate this as a separate API - WDYT about `v3.acls.batch-create`?